client/state/sites/hooks/use-premium-global-styles.ts

```diff
@@ -41,34 +38,18 @@ const getGlobalStylesInfoForSite = ( siteId: number | null ): GlobalStylesStatus
 		.catch( () => DEFAULT_GLOBAL_STYLES_INFO );
 };
 
-export function usePremiumGlobalStyles(): GlobalStylesStatus {
```

Comment:
I really like this change, delegating resolving the site id to the consumer of the hook is a really good idea!